tests/test_prompt_kit_portability.py

```
@@ -0,0 +1,162 @@
+if(JSON.stringify(result.unknown_prompt_ids)!=='["P999"]')throw new Error('unknown preservation');
+const legacy=api.parsePayload('["p07","P03"]');
+if(legacy.schema_version!=='legacy-array/v0')throw new Error('legacy');
+let rejected=false;
+try{api.parsePayload(JSON.stringify({schema_version:'wrong/v1',favorite_prompt_ids:[]}))}catch(error){rejected=true}
+if(!rejected)throw new Error('schema rejection');
+rejected=false;
+try{api.parsePayload(' '.repeat(65537))}catch(error){rejected=true}
+if(!rejected)throw new Error('size rejection');
+console.log('PORTABILITY_RUNTIME_PASS');
+"""
+        completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)